src/renderers/shared/fiber/ReactFiberBeginWork.js

```diff
@@ -296,7 +296,41 @@ module.exports = function<T, P, I, TI, C>(
     if (!coroutine) {
       throw new Error('Should be resolved by now');
     }
-    reconcileChildren(current, workInProgress, coroutine.children);
+
+    const nextChildren = coroutine.children;
```

Comment:
I'd appreciate a comment here mentioning that this code is pretty much the same as `reconcileChildren()`, but operating on `stateNode`.